pkgs/development/python-modules/tuya-device-sharing-sdk/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+, paho-mqtt
+, cryptography
+}:
+let
+  pname = "tuya-device-sharing-sdk";
+  version = "0.2.0";
+in
+buildPythonPackage {
+  inherit pname version;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-fu8zh59wlnxtstNbNL8mIm10tiXy22oPbi6oUy5x8c8=";
+  };
+
+  postPatch = ''
+    touch requirements.txt
+  '';
```

Comment:
Why is that necessary?

Reply:
Please create an issue on the Git repo and reference it here, so we can eventually get rid of this workaround.

Reply:
Do you confirm that the `requirements.txt` file is supposed to be pushed to PyPi?
I'm not very familiar with building python packages (especially about the publication)

Reply:
Anyway I'll do it, just wanted some extra information to put in the issue.